pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -208,13 +208,7 @@ export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
         >
           <Tab
             className={classes.tabLabel}
-            label="Add manually"
-            icon=" "
-            {...a11yProps(0)}
-          />
-          <Tab
-            className={classes.tabLabel}
-            label="ADD FROM GIT"
+            label="ADD FROM SUGGESTIONS"
```

Comment:
This also changes "ADD FROM GIT" to "ADD FROM SUGGESTIONS".
I personally think it is more understandable but let me know your thoughts.

Reply:
Users may confuse at first since there is no directly mention about where those applications configuration be stored, but it's not a big deal. And `from suggestions` mentioned exactly what do we do there, so I think it's good name. Plz also update the docs later

Reply:
👍